addons/mail/static/src/models/composer_view/composer_view.js

```diff
@@ -953,6 +953,7 @@ registerModel({
          */
         composer: many2one('Composer', {
             compute: '_computeComposer',
+            inverse: 'composerViews',
```

Comment:
Not for this PR, but maybe to investigate, it is a bit strange that composer is not an identifying field in composer view, if we follow the usual logic of view components it is not natural. But I guess it is a side issue of composition instead of trait here?

I'm actually wondering in general if inverse fields of `isCausal` shouldn't always be identifying fields on the inverse model. I think that would actually make sense? So pretty much remove `isCausal`, and only make the "inverse of is causal" on all identifying fields?

Would be interesting to list the `isCausal` we have and see which are not actually inverse of identifying fields, and if those could become identifying fields.